src/components/QueueMessage.js

```diff
@@ -50,22 +72,66 @@ class QueueMessage extends React.Component {
       return null
     }
 
+    const handleTabKeyPress = (e, tab) => {
+      if (e.key === 'Enter' || e.key === ' ') {
+        e.stopPropagation()
+        this.onChangeTab(tab)
+      }
+    }
+
     let content
     let button
     if (editing) {
       content = (
-        <>
-          <Input
-            type="textarea"
-            name="text"
-            rows="6"
-            value={editedMessage}
-            onChange={this.onMessageChanged}
-          />
-          <FormText color="muted">
-            You can use Markdown to format this message.
-          </FormText>
-        </>
+        <Card>
+          <CardHeader>
+            <Nav card tabs>
+              <NavItem>
+                <NavLink
```

Comment:
If we set `href="#" on these, we get better visuals when mousing over the tabs. I'm pretty sure if we do this, we'll need to prevent default on the event to avoid having the page hash change.

Reply:
I believe this one is taken care of now (not sure how to check for the page hash changing), mouseover changes the appearance of the pointer